strings/tips/marine.txt

```diff
@@ -13,17 +13,18 @@ There are "specialist" weaponry that can be ordered by Requisitions. However, th
 Anything can be a weapon if you put your mind to it.
 You can check your wounds to see if they have been bandaged or salved by clicking on yourself while in help intent.
 You can check what part of your body is bleeding by shift-clicking on your person.
-Removing weeds as a marine means you can move faster and that xenomorphs cannot take advantages of weeds; your standard boot knife is great for doing this.
-During Self Destruct, green control rods are armed control rods; collect all six!
+Removing weeds as a marine means that xenomorphs cannot take advantages of the battlefield; your standard boot knife is great for doing this.
+Green weeds and purple sticky resin slow you down. Get rid of them!
+During Self Destruct, green control rods are armed control rods. Each rods take 3 minutes to rise, forcing marines to hold Self Destruct for a total of 18 minutes. Collect all six!
 A fit marine can carry many weapons: in their armor, belt, even hanging from their back.
 Requisitions have many supplies to enhance the combat power of marine units, such as attachments, ammunition, and other toys; ask their crew what's on store and you may be pleasantly surprised. DISCLAIMER: Extra gear's availability is dependent on supply.
 Cargo points are valuable: Some crates, like SL armour, standard weaponry or attachments, may be a waste of points which could be better spent on other stuff.
 As a human, you can climb over waist-high obstacles like sandbags, window frames or tables by "SpecialClicking" (Default: CTRL) on them or drag-clicking yourself on them.
-The TX-73 Lasrifle has a mode selector that can be switched using the Unique Action command (Default: Spacebar). Some modes are better than other modes in specific circumstances.
-You can remove armor pieces (leg pieces, arm pieces and chest pieces) and armor modules (Valkyrie, Baldur, etc.) from the XM-02 Combat Exoskeleton using the crowbar and screwdriver espectively.
+The Terra Experimental laser rifle has a mode selector that can be switched using the Unique Action command (Default: Spacebar). Some modes are better than other modes in specific circumstances.
```

Comment:
Nitpicking but I'd suggest making it "The Terra Experimental series of laser rifles have" since unlike the old singular lasgun, now it's multiple weapons.